doc/doxygen/install/install-win.doxygen

```diff
@@ -225,7 +225,11 @@
         \endcode
 
         The CMAKE_PREFIX_PATH should hold the path to your Qt5 build directory (see example below). Note that it is NOT the main Qt5 directory, but the subfolder which is named after the toolchain it was build with (e.g. "5.6/msvc2015_64").
+        @note If you build Qt5 from source, the build directories will be located at <tt><path_to_qt>/mkspecs</tt>. Here you will need to choose the matching directory for your architecture and Visual Studio version.
```

Comment:
should be `<tt>&lt;path-to-qt&gt;</tt>` otherwise Doxygen will try to interpret `<path_to_qt>` as Html and will ignore it.